spyder_env_manager/spyder/widgets/main_widget.py

```diff
@@ -90,11 +90,12 @@ class SpyderEnvManagerWidgetActions:
     ImportEnvironment = "import_environment_action"
     ExportEnvironment = "export_environment_action"
     ToggleExcludeDependency = "exclude_dependency_action"
+    ToggleEnvironmentAsCustomInterpreter = "environment_as_custom_interpreter"
 
 
 class SpyderEnvManagerWidgetOptionsMenuSections:
-    Display = "excludes_section"
-    Highlight = "highlight_section"
+    ImportExport = "import_export_section"
+    EnvironmentDisplayUsage = "environment_display_usage"
```

Comment:
```suggestion
    AdvancedOptions = "advanced_options"
```

Let's rename this section according to my comment before.